index.html

```diff
@@ -2653,7 +2653,7 @@ <h3 id="applying">
               </li>
             </ol>
           </li>
-          <li><a>Trim</a>(<var>value</var>) and convert to lower-case</li>
+          <li><a>Trim</a>(<var>value</var>) and convert <a>to ASCII lowercase</a></li>
```

Comment:
Seems to: https://infra.spec.whatwg.org/#strip-leading-and-trailing-ascii-whitespace

@marcoscaceres what would be the best to use here? 

There is a bug in "dir" as well, where we write

```Otherwise, Trim(value), convert to lower-case, and let direction be the result.```